contracts/script/deploy/router/config/example_config.json

```diff
@@ -1,5 +1,10 @@
 {
     "initialOwner": "0x0000000000000000000000000000000000000001",
-    "initialCertVerifier": "0x0000000000000000000000000000000000000002",
+    "initialRBNs": [
+      0
+    ],
+    "initialCertVerifiers": [
+      "0x0000000000000000000000000000000000000002"
+    ],
```

Comment:
can we update the deployment README explaining how to set these fields?

Reply:
https://github.com/Layr-Labs/eigenda/pull/1787/commits/f264b3f04cc48d9fba5bc86d37bdc7d4bd253178